### PR TITLE
Events

### PR DESCRIPTION
**Pull request by raymondjacobson**
Needs better formatting for how we're displaying events, but it works...

### CODE REVIEW

File: app/helpers/events_helper.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 module EventsHelper
+	def getEventTypes()
```
Comment (dmastylo) [inline app/helpers/events_helper.rb]: just nitpicking but you don't need the parentheses here

**Comment by dmastylo** [comment]
Looks good, wanna write some tests for this?

**Comment by raymondjacobson** [comment]
Yeah, what are we using to write tests? rspec like hartl's thing?

File: app/helpers/events_helper.rb (hunk 1 above)
Comment (raymondjacobson) [inline app/helpers/events_helper.rb]: I'm like the coal in a train. I can ride the rails but know nothing of them.

**Comment by dmastylo** [comment]
Yeah rspec

**Pull request merged**